generated/nirfsg/nirfsg/session.py

```diff
@@ -1840,37 +1831,34 @@ class _SessionBase(object):
-    | ConfigurationSettledEventExportOutputTerm.PXI_TRIG4     | PXI_Trig4   | The trigger is received on PXI trigger line 4.                                                                     |
-    +---------------------------------------------------------+-------------+--------------------------------------------------------------------------------------------------------------------+
-    | ConfigurationSettledEventExportOutputTerm.PXI_TRIG5     | PXI_Trig5   | The trigger is received on PXI trigger line 5.                                                                     |
-    +---------------------------------------------------------+-------------+--------------------------------------------------------------------------------------------------------------------+
-    | ConfigurationSettledEventExportOutputTerm.PXI_TRIG6     | PXI_Trig6   | The trigger is received on PXI trigger line 6.                                                                     |
-    +---------------------------------------------------------+-------------+--------------------------------------------------------------------------------------------------------------------+
-    | ConfigurationSettledEventExportOutputTerm.PXIE_DSTARC   | PXIe_DStarC | The signal is exported to the PXIe DStar C trigger line. This value is valid on only the PXIe-5820/5840/5841/5842. |
-    +---------------------------------------------------------+-------------+--------------------------------------------------------------------------------------------------------------------+
-    | ConfigurationSettledEventExportOutputTerm.TRIG_OUT      | TrigOut     | TRIG IN/OUT terminal.                                                                                              |
-    +---------------------------------------------------------+-------------+--------------------------------------------------------------------------------------------------------------------+
-
-    Note:
-    One or more of the referenced values are not in the Python API for this driver. Enums that only define values, or represent True/False, have been removed.
-    '''
-    exported_done_event_output_terminal = _attributes.AttributeEnum(_attributes.AttributeViString, enums.DoneEventExportOutputTerm, 1150063)
-    '''Type: enums.DoneEventExportOutputTerm
+                **Possible Values**:
+
+    +----------------+--------------------------------------------------------------------------------------------------------------------+
+    | Possible Value | Description                                                                                                        |
+    +================+====================================================================================================================+
+    | empty          | The signal is not exported.                                                                                        |
```

Comment:
It would probably be better to replace this wih a literal empty string.
And I think you should put quotes around your other possible values.